src/promptflow/promptflow/_sdk/_submitter/test_submitter.py

```diff
@@ -38,8 +39,27 @@
 
 
 class TestSubmitter:
-    def __init__(self, flow: Union[ProtectedFlow, EagerFlow], flow_context: FlowContext, client=None):
-        self.flow = flow
+    """
+    Submitter for testing flow/node.
+
+    A submitter will be bonded to a test run (including whether this is a node test or a flow test) after __init__,
+    and will be bonded to a specific executor proxy within an init context:
+    1) we will occupy some resources like a temporary folder to save flow with variant resolved, or an execution
+      service process if applicable;
+    2) output path will also be fixed within an init context;
+    usage:
+    test_submitter = TestSubmitter(...)
+    with test_submitter.init(...) as submitter:
```

Comment:
The reason to support init is that we can init the test_submitter multiple times?

Reply:
The reason to support `init` is that we need a context manager to control the lifecycle of dependent resources, e.g., the execution service; we can call `test_submitter.init` for multiple times and dependent resources will be refreshed in each `init`.

The major reason we haven't made `init` a static method or a class method is to minimize the usage change.